va/va.c

```diff
@@ -927,12 +927,14 @@ VAStatus vaCreateConfig(
     CHECK_DISPLAY(dpy);
     ctx = CTX(dpy);
 
+    VA_TRACE_VVVA(dpy, CREATE_CONFIG, TRACE_BEGIN, profile, entrypoint, num_attribs, attrib_list);
```

Comment:
V short for variable, A short for Array
variable size is depend on data type, i.e profile here is VAProfile, its size is sizeof(VAProfile)
array size is 2d, total size = num of array element x sizeof array element. caller need provide number of array  element, the array element is calc from element data type.

Reply:
What does "VV" stand for?

Reply:
VV means trace carry 2 variable data.